[2021-12-08 22:35:59,399] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.col_process manual__2021-12-08T22:35:52.555303+00:00 [queued]>
[2021-12-08 22:35:59,422] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.col_process manual__2021-12-08T22:35:52.555303+00:00 [queued]>
[2021-12-08 22:35:59,425] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 22:35:59,426] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-08 22:35:59,426] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 22:35:59,448] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): col_process> on 2021-12-08 22:35:52.555303+00:00
[2021-12-08 22:35:59,460] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'col_process', 'manual__2021-12-08T22:35:52.555303+00:00', '--job-id', '548', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpsjg6wfcb', '--error-file', '/tmp/tmpu8k3vr_p']
[2021-12-08 22:35:59,455] {standard_task_runner.py:52} INFO - Started process 4148 to run task
[2021-12-08 22:35:59,463] {standard_task_runner.py:77} INFO - Job 548: Subtask col_process
[2021-12-08 22:35:59,545] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.col_process manual__2021-12-08T22:35:52.555303+00:00 [running]> on host b5fd0372bf6e
[2021-12-08 22:35:59,666] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=col_process
AIRFLOW_CTX_EXECUTION_DATE=2021-12-08T22:35:52.555303+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-08T22:35:52.555303+00:00
[2021-12-08 22:35:59,738] {python.py:152} INFO - Done. Returned value was: None
[2021-12-08 22:35:59,773] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=cpp_workflow, task_id=col_process, execution_date=20211208T223552, start_date=20211208T223559, end_date=20211208T223559
[2021-12-08 22:35:59,843] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-08 22:35:59,905] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
